Remove unused transparent box

x5hwuk7nc6jsudcc opened the pull request.

davidgatti reviewed (changes requested):
From your PR I see that you resized the image while not removing the white transparent bag-round under the image. Please remove the box-shadow as market in the image bellow from the CSS. and keep the rest as it is from the master branch.

![Screen Shot 2021-06-02 at 7 52 42 AM](https://user-images.githubusercontent.com/411114/120430647-9c8ddf00-c377-11eb-9a53-b27b10bba7ec.png)

x5hwuk7nc6jsudcc commented:
> From your PR I see that you resized the image while not removing the white transparent bag-round under the image. Please remove the box-shadow as market in the image bellow from the CSS. and keep the rest as it is from the master branch.

Hi @davidgatti I've removed the class from the image which will remove the white background.  I haven't reverted the image back yet as just want to confirm you want this change? The image itself had a white background, which I removed, as well as re-sizing it to properly fit that container.  Let me know if you want me to revert back to original, otherwise this PR is ready to merge.
